### PR TITLE
Add meta role label

### PR DESCRIPTION
**Pull request by Labbs**
Add meta role label like : __meta_kubernetes_node_role="worker"

### CODE REVIEW

File: discovery/kubernetes/node.go [hunk 1]
```diff
@@ -140,16 +141,31 @@ func nodeSourceFromName(name string) string {
...
+	rolePrefix = "node-role.kubernetes.io/"
+	roleLabel  = "kubernetes.io/role"
 )
 
 func nodeLabels(n *apiv1.Node) model.LabelSet {
 	ls := make(model.LabelSet, len(n.Labels)+len(n.Annotations)+1)
 
 	ls[nodeNameLabel] = lv(n.Name)
 
+	for k, v := range n.Labels {
```
Comment (brian-brazil) [inline discovery/kubernetes/node.go]: We already have this under node_label as far as I can see.

Comment (Labbs) [reply discovery/kubernetes/node.go]: yes exactly, i change that

Comment (brian-brazil) [reply discovery/kubernetes/node.go]: My point is that this is already supported and available to relabelling, so your change isn't needed.

Comment (Labbs) [reply discovery/kubernetes/node.go]: yes when we use `kubernetes.io/role="worker"`, it's simple.
But with `node-role.kubernetes.io/worker=""`, how would you do with this syntax? I tried different things without success.

Comment (simonpasquier) [reply discovery/kubernetes/node.go]: Hmm, I suspect that you're hitting https://github.com/prometheus/prometheus/issues/3726.

**Comment by roidelapluie** [comment]
We have decided to close this PR during our bug scrub per above comments.

**Pull request closed without merging**